docs/getting-started.md

```diff
@@ -56,14 +55,20 @@ curl localhost:10101/schema
 {"indexes":null}
```

Comment:
We should update this to show a non-null response - maybe a response after importing is complete, and a note explaining that.